bins/svc/preprocess.py

```diff
@@ -85,7 +85,7 @@ def preprocess(cfg, args):
             cfg.dataset_path[dataset],
             output_path,
             cfg.preprocess,
-            is_custom_dataset=cfg.use_custom_dataset,
+            is_custom_dataset=dataset in cfg.use_custom_dataset,
```

Comment:
what does mean in this line?

Reply:
The code before is a bit buggy: the `use_custom_dataset` field in config was initially a bool value, which will make **every dataset be processed as custom dataset** when setting it `true`. So it cannot work properly when users want to use both existing and custom dataset.

The new code turn `use_custom_dataset` field into a list which only stores custom dataset name, so it can handle both existing and custom dataset. I've modified everything related to this config to follow this setting and also specified it in the MultipleContentsSVC readme file. We'll add this across other tasks/models in the future.